### ISSUE TITLE
spaCy v0.80 python3 incompatibility

### ISSUE THREAD

**Issue by NSchrading** [honnibal/spaCy#50]
In the new multi_words.py RegexMerger, there is a reference to unicode(tokens). In Python 3, there is no "unicode()" function, so it causes spaCy to crash.

```python
from spacy.en import English
nlp = English()
tok = nlp("test test test")
```
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/spacy/en/__init__.py", line 195, in __call__
    self.mwe_merger(tokens)
  File "/spacy/multi_words.py", line 7, in __call__
    for m in regex.finditer(unicode(tokens)):
NameError: name 'unicode' is not defined
```

**Comment by honnibal** [comment]
Damn, thanks. Rushed a fix out; try v0.81, which should be on pip now.

Travis CI isn't working for me anymore, because I need too much memory. My jobs keep getting randomly killed. I'll set up my own CI server with Jenkins when I have time.

**Comment by NSchrading** [comment]
Yep, I think that fixed it. Thanks!
